tests/docker-compose.yml

```diff
@@ -1,36 +1,30 @@
-version: '3.2'
-
-# Only one of these will be up at a time for now.
-# Concurrency will be on the tests folder we have.
-
 services:
   modsec2-apache:
     container_name: modsec2-apache
     image: owasp/modsecurity-crs:apache
     environment:
-      SERVERNAME: modsec2-apache
+      ACCESSLOG: "/var/log/apache2/access.log"
       BACKEND: http://backend
-      PORT: "80"
-      MODSEC_RULE_ENGINE: DetectionOnly
-      PARANOIA: 4
-      TZ: "${TZ}"
-      ERRORLOG: "/var/log/error.log"
-      ACCESSLOG: "/var/log/access.log"
+      COMBINED_FILE_SIZES: "65535"
+      CRS_ENABLE_TEST_MARKER: 1
+      ERRORLOG: "/var/log/apache2/error.log"
```

Comment:
The workflow creates `/var/log/modsec2-apache/apache2`.

Reply:
Hmmm... it should not, volumes say:
```
     volumes:
       - ./logs/modsec2-apache:/var/log/apache2:rw
```

Reply:
Wait. Got it.